offix/src/main/java/org/aerogear/offix/interceptor/OffixInterceptor.kt

```diff
@@ -0,0 +1,37 @@
+package org.aerogear.offix.interceptor
+
+import android.util.Log
+import com.apollographql.apollo.api.Mutation
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import com.apollographql.apollo.interceptor.ApolloInterceptorChain
+import java.util.concurrent.Executor
+
+class OffixInterceptor : ApolloInterceptor {
```

Comment:
Offline?

Reply:
@wtrocki What do you mean here by Offline? Are you referring to changing the name of the class or something related to offline support?

Reply:
Yep, let's call it ConflictInterceptor instead.